posthog/tasks/usage_report.py

```diff
@@ -1219,6 +1221,24 @@ def _get_full_org_usage_report_as_dict(full_report: FullUsageReport) -> dict[str
     return dataclasses.asdict(full_report)
 
 
+def _queue_report(producer, organization_id: str, full_report_dict: dict[str, Any]) -> None:
+    logger.info(f"Sending usage report for organization {organization_id}")  # noqa T201
+
+    json_data = json.dumps(
+        {"organization_id": organization_id, "usage_report": full_report_dict}, separators=(",", ":")
+    )
+
+    compressed_bytes = gzip.compress(json_data.encode("utf-8"))
+
+    response = producer.send_message(
+        message_body=compressed_bytes, content_encoding="gzip", content_type="application/json"
```

Comment:
don't content encoding/type need to be under message_attributes?

Reply:
yep, fixing